core/src/main/java/com/scalar/db/storage/common/checker/OperationChecker.java

```diff
@@ -36,7 +36,7 @@ public void check(Get get) throws ExecutionException {
     checkProjections(get, metadata);
 
     if (Utility.isSecondaryIndexSpecified(get, metadata)) {
-      if (!new ColumnChecker(metadata, true).check(get.getPartitionKey().get().get(0))) {
+      if (!new ColumnChecker(metadata, false).check(get.getPartitionKey().get().get(0))) {
```

Comment:
Was it a bug?

Reply:
Yes, I found we needed to accept empty values here, that's a very rare case though.